frontend/src/Interfaces/PopupBoxProps.tsx

```diff
@@ -0,0 +1,9 @@
+import { MouseEventHandler } from "react";
+
+export default interface PopupBoxProps {
+  title: string | undefined;
```

Comment:
compulsory fields shouldn't be nullable

plus, nullable field should be indicated with:
```
export interface SomeProps {
  nullableField?: string
}
```